.gitignore

```diff
@@ -37,3 +37,4 @@ i18n
 /.cproject
 /.project
 frontend/ui/screensaver.lua~
```

Comment:
Remove that too specific one, and may be move the next one `*.lua~` above around line 9 near its buddies.